src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -76,28 +78,42 @@ class Comment extends React.Component<Props, State> {
         isInputOpen: false,
     };
 
+    selectComment = (isSelected: boolean) => {
```

Comment:
Can we swap `ActivityCard` for `SelectableActivityCard` to avoid duplication and keep the logic consistent between annotation and comment cards?

Reply:
No, I don't think this would work for the design we want to achcive. `ActivityCard` is too low in component tree. Actually, I think we don't need `isSelected` on `ActivityThread` we can do it with `isFocused` and reuse `activeEntry` as a single source of truth for selected thread.

Reply:
the login in SelectableActivityCard isn't matching what we want to have here, but we will need to touch it and change to `SelectableActivityThread` to align Annotations to new design and having threads. We decided to not do it in this PR as it's taking quite some time now. And do a follow up ticket for it